### PR TITLE
To-Do-List 

### PR DESCRIPTION
**Pull request by atatm**
- [ ] set up linters
- [ ] set up webpacks
- [ ] Add todo list that fullfill the requirment
- [ ] CSS is loaded by Webpack Style/CSS Loader.
changes after the feedback
- [ ]  update the readme like the code reviewer commented.
- [ ] add the objects that take those keys like on the instruction.

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @atatm 
I can't proceed to review this project because you have submitted an entire application in one PR, kindly build the project milestone by milestone; each milestone needs to be built in a corresponding branch; at the end of the project, you should have as many feature-branches as milestones, the PRs should be used when asking for a review in the dashboard corresponding to the milestone the student worked on.

Kindly stick to the first milestone for this PR and submit it for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+To run this project locally, make sure you have done the following:
+
+- Created hint.rc file
+- Created stylelint.rc file
+- Created .gihub/workflows/linters/yml
+- Get a copy of the project by running `https://github.com/atatm/To-Do-list.git `
+- cd into the project
+- Open the project using code editor of your choice
+- Open your prefered browser
```
Comment (ha-manel) [inline README.md:27]: [Optional]

- [ ] It's better if you add how to run the project using `npm`. Ex: run `npm install` and `npm start` etc...

File: README.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+
+- LinkedIn: [Alelign Ayana](https://linkedin.com/@alelignayana)
+
+## :handshake: Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+### Go Live
+
+https://atatm.github.io/To-Do-list/
```
Comment (ha-manel) [inline README.md]: - [ ] Kindly add `/dist/index.html` to this link so it becomes a live demo to the app, right now it's taking the users to your README file 👍

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+import './style.css';
```
Comment (ha-manel) [inline src/index.js:1]: - [ ] As mentioned in the requirements, kindly add some hardcoded tasks to your list. You can do that by creating an array of a few objects where each object has at least these keys: `index,` `description` and `completed`. Then, create a function that iterates over the tasks array and populates the HTML list item element for each task.

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @atatm 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️

## Required Changes:
_Kindly check the comments below_

## Optional:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Abenezer-Tilahun) [reply README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (Abenezer-Tilahun) [reply README.md:27]: - [ Optional ] Kindly Address the previous reviewer suggestion about  how to run the project using `npm`

File: src/index.js (hunk 3 above)
Comment (Abenezer-Tilahun) [reply src/index.js:1]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @atatm     
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Good README  
✔️ Good Commit Messages



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by atatm** [comment]
thank you

**Pull request merged**